src/test/regress/expected/multi_subquery_behavioral_analytics.out

```diff
@@ -0,0 +1,512 @@
+--
+-- multi subquery behavioral analytics queries aims to expand existing subquery pushdown
+-- regression tests to cover more cases
+-- the tables that are used depends to multi_insert_select_behavioral_analytics_create_table.sql
+--
+-- We don't need shard id sequence here, so commented out to prevent conflicts with concurrent tests
+-- ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1430000;
+-- ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1430000;
+ 
+SET citus.subquery_pushdown TO TRUE;
+SET citus.enable_router_execution TO FALSE;
+------------------------------------
+-- Vanilla funnel query
+------------------------------------
+SELECT user_id, array_length(events_table, 1)
+FROM (
```

Comment:
I really love that we added these regression tests for subquery pushdown.

However, looking into the tests, they are mostly not relevant to the changes (i.e., top level subquery includes a single target entry). So, could you add some regression tests that have top level joins? For instance, mostly using the same tests but removing the top level aggregations?


For instance, removing the top level aggregations from the next test I got an error (even if I remove the UNION and use a single query on the LEFT of the join tree):
```SQL
SELECT *
  FROM (
    (
      SELECT u.user_id, 'step=>1'::text AS event, e.time
      FROM users_table AS u,
          events_table AS e
      WHERE  u.user_id = e.user_id
      AND u.user_id >= 10
      AND u.user_id <= 25
      AND e.event_type IN (100, 101, 102)
    )
    UNION
    (
      SELECT u.user_id, 'step=>2'::text AS event, e.time
      FROM users_table AS u,
         events_table AS e
      WHERE  u.user_id = e.user_id
      AND u.user_id >= 10
      AND u.user_id <= 25
      AND e.event_type IN (103, 104, 105)
    )
  ) t1 LEFT JOIN (
      SELECT DISTINCT user_id, 
        'Has done event'::TEXT AS hasdone_event
      FROM  events_table AS e
      
      WHERE  e.user_id >= 10
      AND e.user_id <= 25
      AND e.event_type IN (106, 107, 108)

  ) t2 ON (t1.user_id = t2.user_id);
```

I got the following error:
```
ERROR:  bogus varno: 3
```